src/ssl.c

```diff
@@ -271,6 +273,325 @@ const WOLF_EC_NIST_NAME kNistCurves[] = {
 };
 #endif
 
+#if defined(HAVE_HPKE) && defined(HAVE_ECC)
+int wolfSSLSetEchConfigs64(WOLFSSL* ssl, char* echConfigs64,
+    word32 echConfigs64Len)
+{
+    int ret;
+    word32 decodedLen = echConfigs64Len * 3 / 4 + 1;
```

Comment:
Can we get this from Base64_Decode call?

Reply:
Base64_Decode does not have a LENGTH_ONLY_E return if you give it a null out and I need to allocate the buffer before calling